analysis/results/iso_distributions/iso_distribution.all.csv

```diff
@@ -0,0 +1,59 @@
+cloud,region,iso
```

Comment:
Do the files in `results/iso_distributions` take a long time to generate? If not, we probably don't need to check them in.

Also, it seems redundant to have each cloud in a separate file and also a combined file. Can we just use `iso_distribution.all.csv`? 

Lastly, we also need to have some kind of ground truth for the physical location (probably in lat/lon or a city name) of each datacenter. I'm wondering if we should have another csv with that information or if we can combine it into these csvs.

Reply:
No, they don't, but the results need manual inspection and format change into CSVs and they will almost never change, so I'm checking in the file to facilitate automated scripting of the entire process.

For redundancy, that is correct. I initially had separate files, but later on I need a unified file, so I can delete the two separate ones, but keeping the combined CSV will make it easier to run the later steps unmodified as in the README.

That's a good point. I think we can use the same file (with a different name), or different files (`cloud,region,gps`). Let me raise another PR when that's ready.